server/src/engineModel/EngineModel.ts

```diff
@@ -51,18 +51,19 @@ export class EngineModel {
     }
     private buildYmlFunction(func: any, sourceType?: string): AbstractYmlFunction {
         let ymlAbstractFunction: AbstractYmlFunction;
+        const documentation: string = !!func.doc ? func.doc[0] : null;
         if (sourceType) {
-            ymlAbstractFunction = new YmlFunction(func.$.ident, this.uri);
+            ymlAbstractFunction = new YmlMethod(func.$.ident, this.uri);
             this.functions.push(ymlAbstractFunction);
-            ymlAbstractFunction.detail = `[${sourceType}].${ymlAbstractFunction.label}`;
+            ymlAbstractFunction.setUserInformations(
+                `(function) [${sourceType}].${ymlAbstractFunction.label}`,
+                documentation,
+            );
         } else {
-            ymlAbstractFunction = new YmlMethod(func.$.ident, this.uri);
-            ymlAbstractFunction.detail = `[STATIC] ${ymlAbstractFunction.label}`;
+            ymlAbstractFunction = new YmlFunction(func.$.ident, this.uri);
+            ymlAbstractFunction.setUserInformations(`(function) [STATIC] ${ymlAbstractFunction.label}`, documentation);
```

Comment:
Strange that “function” is all quite while “STATIC” yells like crazy. (other occurrences elsewhere)